openstates/scrape/base.py

```diff
@@ -138,8 +145,33 @@ def save_object(self, obj):
         self.output_names[obj._type].add(filename)
 
         if self.scrape_output_handler is None:
-            with open(os.path.join(self.datadir, filename), "w") as f:
-                json.dump(obj.as_dict(), f, cls=utils.JSONEncoderPlus)
+
+            file_path = os.path.join(self.datadir, filename)
+
+            # Remove redundant prefix
+            try:
+                file_path_ = file_path[file_path.index("_data") + 6 :]
+            except Exception:
+                file_path_ = file_path
+
+            if self.realtime:
+
+                s3 = S3FileSystem(anon=False)
+
+                S3_FULL_PATH = settings.S3_REALTIME_BASE + str(file_path_)
```

Comment:
This is another string formatting pattern, different than the two above. JS-style str concats are _also_ fine, but we should be consistent.